x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsIntegTests.java

```diff
@@ -463,7 +463,9 @@ public void testMaxRestoreBytesPerSecIsUsed() throws Exception {
             // we compute the min across all the max shard sizes by node in order to
             // trigger the rate limiter in all nodes. We could just use the min shard size
             // but that would make this test too slow.
-            long rateLimitInBytes = getMaxShardSizeByNodeInBytes(indexName).values().stream().min(Long::compareTo).get();
+            // Update: setting rate limit at a 90% of minimax value above
```

Comment:
We should add a small explanation about why we're using the 90% of the minmax value to set the rate limiter.
Also, we rely on the version control system to track updates.

Reply:
I think it is already easy to track back to this PR and this writing by looking on commit history. Would you like to include link on this PR to the comments?

Reply:
I usually prefer to have all the context in the code, just a small comment would be great 👍